From 7c016f8bfeff0ed658fef66edde755f464dd5b8b Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Fri, 20 Nov 2020 10:17:18 +0800
Subject: [PATCH] arm64: dts: rockchip: rk356x: remove keep-power-in-suspend
 from sdhci

sdhci aims for emmc but keep-power-in-suspend is for sdio devices.

Change-Id: If70e8d450df607811149a580c273ebcdfb9e3592
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-k108.dts     | 1 -
 arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-rkg11.dts    | 1 -
 arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts          | 1 -
 arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi                  | 1 -
 arch/arm64/boot/dts/rockchip/rk3568-iotest-ddr3-v10-linux.dts | 1 -
 arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi                  | 1 -
 6 files changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-k108.dts b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-k108.dts
index 569f28b68e8a..cdab8b281459 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-k108.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-k108.dts
@@ -1137,7 +1137,6 @@
 	bus-width = <8>;
 	supports-emmc;
 	non-removable;
-	keep-power-in-suspend;
 	rockchip,txclk-tapnum = <0x10>;
 	max-frequency = <200000000>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-rkg11.dts b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-rkg11.dts
index 18115bc11289..84b4bab4fbed 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-rkg11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet-rkg11.dts
@@ -1038,7 +1038,6 @@
 	bus-width = <8>;
 	supports-emmc;
 	non-removable;
-	keep-power-in-suspend;
 	rockchip,txclk-tapnum = <0x10>;
 	max-frequency = <200000000>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts
index bba3b1866e9b..041633662ab7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts
@@ -1060,7 +1060,6 @@
 	bus-width = <8>;
 	supports-emmc;
 	non-removable;
-	keep-power-in-suspend;
 	rockchip,txclk-tapnum = <0x10>;
 	max-frequency = <200000000>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
index 14ff21082cda..c32b14127b24 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
@@ -1607,7 +1607,6 @@
 	bus-width = <8>;
 	supports-emmc;
 	non-removable;
-	keep-power-in-suspend;
 	rockchip,txclk-tapnum = <0x10>;
 	max-frequency = <200000000>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/rk3568-iotest-ddr3-v10-linux.dts b/arch/arm64/boot/dts/rockchip/rk3568-iotest-ddr3-v10-linux.dts
index 3f09dc5d52c2..46db54722b66 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-iotest-ddr3-v10-linux.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3568-iotest-ddr3-v10-linux.dts
@@ -39,7 +39,6 @@
 	bus-width = <8>;
 	supports-emmc;
 	non-removable;
-	keep-power-in-suspend;
 	rockchip,txclk-tapnum = <0x10>;
 	max-frequency = <200000000>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi
index 543d6f1e59bb..1495829d2171 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi
@@ -316,7 +316,6 @@
 	bus-width = <8>;
 	supports-emmc;
 	non-removable;
-	keep-power-in-suspend;
 	status = "okay";
 };
 
-- 
2.35.3

